src/main/perl/Process.pm

```diff
@@ -153,10 +177,125 @@ C<<<msg> command: <COMMAND>[ <postmsg>]>>.
 
 =cut
 
+sub _get_uid_gid
+{
+    my ($self, $mode) = @_;
+
+    my @res;
+    my $what = $self->{$mode};
```

Comment:
I find $what to be a bit vague. How about $targetid ?

Reply:
i'll use `target` (the id part suggests only ids are used, while i think most likely people will use usernames instead of ids.